src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -2279,12 +2279,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         hasErrors = True
                     Else
                         Dim useSiteDiagnostics As HashSet(Of DiagnosticInfo) = Nothing
+                        Dim accessThroughType = GetAccessThroughType(actualEventAccess.ReceiverOpt)
+                        If Not (Me.IsAccessible(method, useSiteDiagnostics, accessThroughType) OrElse
+                            Not IsAccessible(eventSymbol, useSiteDiagnostics, accessThroughType)) Then
```

Comment:
Why are we checking both the `method` and `eventSymbol` here for accessibility? I don't believe that VB events can declare custom accessibility hence I would assume that we should be checking only one here.

@cston ?